modin/core/execution/ray/common/utils.py

```diff
@@ -163,12 +166,7 @@ def initialize_ray(
     runtime_env_vars = ray.get_runtime_context().runtime_env.get("env_vars", {})
     for varname, varvalue in env_vars.items():
         if str(runtime_env_vars.get(varname, "")) != str(varvalue):
-            if is_cluster or (
-                # Here we relax our requirements for a non-cluster case allowing for the `env_vars`
-                # to be set at least as a process environment variable
-                not is_cluster
-                and os.environ.get(varname, "") != str(varvalue)
```

Comment:
@dchigarev Now all environment variables are set temporarily before Ray is initialized (in case of a local cluster), so this check will not work. This seems to be enough for Ray's processes to start using these variables.

I don't see any particular need to use these environment variables for the main process either. What do you think?

Reply:
agree